include/RAJA/scan.hxx

```diff
@@ -106,10 +116,12 @@ void exclusive_scan_inplace(Iter begin, Iter end, BinaryFunction f, T v)
 
 template <typename ExecPolicy,
           typename Iter,
-          typename T = typename std::iterator_traits<Iter>::value_type>
+          typename T = typename std::iterator_traits<Iter>::value_type,
+          typename Defaults = typename scan::defaults<ExecPolicy, T>,
+          typename BinaryOp = typename Defaults::binary_op>
```

Comment:
I can leave the typename chain for Defaults up here or move them into the body of the function. I'm open to suggestions.